ibm/data_source_ibm_iam_role_actions.go

```diff
@@ -71,6 +77,15 @@ func datasourceIBMIAMRoleActionRead(d *schema.ResourceData, meta interface{}) er
 	d.Set("manager", flattenActionbyDisplayName("Manager", serviceRoles))
 	d.Set("reader_plus", flattenActionbyDisplayName("ReaderPlus", serviceRoles))
 	d.Set("writer", flattenActionbyDisplayName("Writer", serviceRoles))
+	d.Set("actions", flattenRoleActions(serviceRoles))
 
 	return nil
 }
+
+func flattenRoleActions(object []iampolicymanagementv1.Role) map[string]string {
+	actions := make(map[string]string)
+	for _, item := range object {
+		actions[*item.DisplayName] = strings.Join(item.Actions, ",")
```

Comment:
@kavya498 do you see any benefit in adding a condition before this line as follows:
```
if !(*item.DisplayName == "Reader" || *item.DisplayName == "Manager" || *item.DisplayName == "ReaderPlus" || *item.DisplayName == "Writer") {
	actions[*item.DisplayName] = strings.Join(item.Actions, ",")
}
```
So that we don't duplicate data for Reader, Manager, Writer, ReaderPlus which is already present in the other data source attributes?

Reply:
@guicabr , I think this should be fine..
Let `actions` attribute give all the actions no matter what role it is..